erica/request_processing/erica_input/v2/erica_input.py

```diff
@@ -41,10 +40,18 @@ class ErrorRequestQueue(BaseModel):
     errorMessage: str
 
 
+class Status(Enum):
+    PENDING = "Pending"
+    PROCESSING = "Processing"
+    FAILED = "Failed"
+    COMPLETED = "COMPLETED"
+
+
 class ResponseGetFromQueue(BaseModel):
```

Comment:
Shouldn't we then somehow specify that we will also use this model for successful cases?

Reply:
I will name all response models for the HTTP 200 like "SuccessResponse..."